### ISSUE TITLE
IMG file with a encrypted disk

### ISSUE THREAD

**Issue by motolese** [mynodebtc/mynode#642]
In the last release 2.5.3, mynode team included the IMG disk image, allowing users to run it on a physical drive on a PC. 

For security reasons would be great, if we had the same image as an encrypted disk, using the default password 'bolt' turning easier to change the encryption password later. 

Also, before format, would be great to have the same functionality for the external drive.

Best wishes!

**Comment by as-schug** [comment]
Yeah... Data disk partition could be encrypted and user should enter password on every system start. In could be a great feature.
